packages/open-payments-client/src/index.ts

```diff
@@ -0,0 +1,64 @@
+import { components } from './types'
+import axios, { AxiosInstance } from 'axios'
```

Comment:
considered universal/isomorphic fetch ? 
node 18 LTS is due very soon
https://dev.to/cloudx/nodejs-18-fetch-api-test-runner-module-and-more-2ckg

Reply:
Yes, looks to be out Oct 25th: https://github.com/nodejs/release#release-schedule. I have `axios` for now as I was mirroring the existing `OpenPaymentsClientService` https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/client/service.ts#L153-L180. I know we use `axios` in a few places so maybe have a team discussion whether we want to upgrade node and replace `axios` with `fetch` altogether once node 18 is out